tests/textcase/plugin/plugin_spec.lua

```diff
@@ -156,6 +156,19 @@ describe("plugin", function()
           "elit-sed dolor-sit amet",
         },
       },
+      {
+        keys = "<C-V>jjEE<CMD>Subs/lorem_ipsum/elit_sed/<CR>",
```

Comment:
Is it required to `jjEE`? I think Subs functionality is buggy, wondering if it fails when the CMD is executed with the cursor located at the beginning of the line

Reply:
This one is not failing. It was just as an example for the issue.

The case you are describing has been fixed by the other visual selection bugfix.